garak/buffs/art.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+""" Buff that converts prompts with different encodings. """
+
+from collections.abc import Iterable
+
+import logging
+import garak.attempt
+from garak.buffs.base import Buff
+from art import *
```

Comment:
Some disprefer `import *` because it obfuscates exactly which items are added to the current namespace

```suggestion
from art import text2art
```